src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.html

```diff
@@ -32,6 +32,28 @@ <h1 mat-dialog-title>GitHub App Logs</h1>
           <th mat-header-cell *matHeaderCellDef mat-sort-header>GitHub Username</th>
           <td mat-cell *matCellDef="let element">{{ element.githubUsername }}</td>
         </ng-container>
+        <ng-container matColumnDef="entryName">
+          <th mat-header-cell *matHeaderCellDef mat-sort-header>Entry Name</th>
+          <td mat-cell *matCellDef="let element">{{ element.entryName ? element.entryName : 'n/a' }}</td>
+        </ng-container>
+        <ng-container matColumnDef="deliveryId">
+          <th mat-header-cell *matHeaderCellDef mat-sort-header>Delivery ID</th>
+          <td mat-cell *matCellDef="let element">
+            <div fxLayout fxLayoutAlign="start center">
+              <span class="w-100">{{ element.deliveryId.substring(0, 7) }}</span>
+              <button
+                mat-icon-button
+                color="secondary"
+                type="button"
+                matTooltip="Copy Delivery ID"
```

Comment:
You could show the full delivery id in the tooltip.

Reply:
With the caveat that copying tooltips is probably not reliable/common